test/unittest/rtps/builtin/BuiltinDataSerializationTests.cpp

```diff
@@ -129,6 +129,86 @@ TEST(BuiltinDataSerializationTests, ok_with_defaults)
     }
 }
 
+TEST(BuiltinDataSerializationTests, msg_without_datasharing)
+{
+    {
+        uint8_t data_r_buffer[] =
+        {
+            // Encapsulation
+            0x00, 0x03, 0x00, 0x00
+        };
+
+        CDRMessage_t msg(0);
+        msg.init(data_r_buffer, static_cast<uint32_t>(sizeof(data_r_buffer)));
+        msg.length = msg.max_size;
+
+        ReaderProxyData out(max_unicast_locators, max_multicast_locators);
+        out.readFromCDRMessage(&msg, network, false, true);
```

Comment:
In `2.6.x` I think that this method only accepts 3 arguments 
```suggestion
        out.readFromCDRMessage(&msg, network, false);
```